analyzer/codechecker_analyzer/cmd/parse.py

```diff
@@ -390,6 +390,22 @@ def add_arguments_to_parser(parser):
                              "containing analysis results which should be "
                              "parsed and printed.")
 
+    parser.add_argument('--config',
+                        dest='config_file',
+                        required=False,
+                        help="R|Allow the configuration from an "
+                             "explicit JSON based configuration file. "
+                             "The value of the 'parse' key in the "
+                             "config file will be emplaced as command "
+                             "line arguments. The format of "
+                             "configuration file is:\n"
+                             "{\n"
+                             "  \"parse\": [\n"
+                             "    \"--trim-path-prefix\",\n"
+                             "    \"/home/workspace\"\n"
+                             "  ]\n"
+                             "}")
```

Comment:
```suggestion
                             "}.\n"
                             "You can use any environment variable "
                             "inside this file and it will be "
                             "expaneded.")
```